include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -1423,15 +1436,15 @@ namespace internal
    * derivative through collocation techniques.
    *
    * This class allows for dimension-independent application of the operation,
-   * implemented by template recursion. It has been tested up to 6D.
+   * implemented by template recursion. It has been tested up to 6d.
    */
   template <EvaluatorVariant  variant,
             EvaluatorQuantity quantity,
             int               dim,
             int               basis_size_1,
             int               basis_size_2,
-            typename Number,
-            typename Number2>
+            typename = bool,
+            typename = bool>
```

Comment:
Do you want to add a comment here, why you did this? Just as you did in the description of the PR?